styles/components/_select2.scss

```diff
@@ -22,3 +22,72 @@
     }
   }
 }
+
+// Tags (the element that select2 is generated off of must have a class of 'select2Tags').
+// Focus styling is with other form focus styling in form.scss.
+.select2TagsWrap {
+  position: relative;
+
+  .select2Tags + .select2-container {
+    &.select2-container--default {
+      .select2-selection--multiple .select2-selection__rendered {
+        padding-bottom: $padSm;
+        border-radius: 3px;
+      }
```

Comment:
In my new branch, for Shipping, I've added a height 100% to a certain element that has fixed this issue. So, this will be fixed in a future PR.